cluster/manifests/deletions.yaml

```diff
@@ -4,5 +4,14 @@
 #   namespace: kube-system
 #   kind: deployment
 
-pre_apply: [] # everything defined under here will be deleted before applying the manifests
+pre_apply: # everything defined under here will be deleted before applying the manifests
```

Comment:
wouldn't this leave us without any DNS during the cluster update procedure? (old kube-dns deleted, new CoreDNS not applied yet)

Reply:
> If we want to switch the idea would be to roll it out as a replacement for one of the kube-dns instances and then replace the kube-dns-b instance later on. This way we should have no downtime for dns.

@hjacobs maybe you are not aware, but we run two instances of kube-dns to have a fallback for each pod in the `relsolv.conf.` We have DNS servers on `10.3.0.10` and `10.3.0.11`. This will only delete the first one, so traffic should go to the second while coredns is being applied.